lib/twilio/rtc/peerconnection.ts

```diff
@@ -550,7 +550,9 @@ PeerConnection.prototype._reassignMasterOutput = function reassignMasterOutput(p
   pc.outputs.delete(masterId);
 
   const self = this;
-  const idToReplace = Array.from(pc.outputs.keys())[0] || 'default';
+  const activeDevice = Array.from(pc.outputs.keys())[0];
+  // The audio device key could also be '' on Chrome if no media device permissions are allowed
+  const idToReplace = activeDevice !== null && activeDevice !== undefined ? activeDevice : 'default';
```

Comment:
Checking against `undefined` might be enough if guarding against `outputs` being empty?